run.sh

```diff
@@ -40,7 +40,7 @@ _set_static_routes() {
             interface=${ifname}
         fi
         for route in ${STATIC_ROUTES}; do
-            ip route add ${route} dev "${interface}" || true
+            eval "ip route add ${route} dev ${interface}" || true
```

Comment:
just removing the quotes around ${interface} should be enough. there is no reason for evil eval.

Reply:
the quotes around the interface are not the problem.